src/main/scala/org/biodatageeks/sequila/pileup/PileupMethods.scala

```diff
@@ -38,6 +38,7 @@ object PileupMethods {
 
     //FIXME: Add automatic unpersist
     val aggregates = ContigAggrTimer.time {alignmentsInstr.assembleContigAggregates.persist(storageLevel) }
+    aggregates.setName(InternalParams.RDDEventsName)
```

Comment:
w InternalParams dodalabym nowy parametr RDDAggregatesName. Poniewaz ten parametr co uzyles odnosi sie do eventow z pokrycia, a te agregaty to duzo wiecej i imho powinno byc rozdzielone.

Reply:
Rozdzielilem jak sugerowalas.